content/docs/command-reference/list.md

```diff
@@ -62,6 +61,7 @@ accessed with `dvc get`, `dvc import`, or `dvc.api`.
   [Git revision](https://git-scm.com/docs/revisions)) of the repository to list
   content for. The latest commit in `master` (tip of the default branch) is used
   by default when this option is not specified.
+- `--show-json` - show the output in JSON format.
```

Comment:
```suggestion
- `--show-json` - show the output in JSON format.
```
```suggestion
- `--show-json` - prints the command's output in easily parsable JSON format,
  instead of a human-readable table.
```